switchsdk/src/androidTest/java/net/gini/switchsdk/uitests/ReviewPictureScreenTest.java

```diff
@@ -40,8 +36,8 @@
 
 @RunWith(AndroidJUnit4.class)
 @LargeTest
-public class ReviewPictureScreenTest {
-    private static final int TIME_TO_WAIT_BETWEEN_STEPS = 1000;
+public class ReviewPictureScreenTest extends CommonUiTests {
```

Comment:
I would have preferred using composition, rather than inheritance. Or at least, if `CommonUiTests ` would be a super class, `BaseUITests` might have been a better name.